pkg/scheduler/scheduler.go

```diff
@@ -57,6 +57,8 @@ const (
 	SchedulerError = "SchedulerError"
 	// Percentage of plugin metrics to be sampled.
 	pluginMetricsSamplePercent = 10
+	// Duration the scheduler will wait before expiring an assumed pod.
+	durationToExpireAssumedPod = 15 * time.Minute
```

Comment:
Let's add a comment here explaining the motivation, like a short duration may lead to false negative to invalidate the pod too soon, and hence cause racing conditions...

Reply:
the motivation for increasing the timeout is in the issue, not sure if it makes sense to say that it was 30s and then we increased it to 15min in the code.

Reply:
I added a link to the issue.